mkdocs.yml

```diff
@@ -36,7 +36,7 @@ nav:
     - Register packages on-chain: 'guides/register_packages_on_chain.md'
     - Deploy a service: 'guides/deploy_service.md'
     - Running a service in different networks: 'guides/running_service_different_networks.md'
-    - Utilising service overrides: 'guides/service_level_overrides.md'
+    - The service definition file: 'guides/service_definition_file.md'
```

Comment:
```suggestion
    - The service component: 'guides/service_definition_file.md'
```

Reply:
I have a minor complain: the "service component" refers to the collection of all items that compose a service (typically just the yaml file and the README.md, but in theory it could be more files...)
However, in this section we are just describing one file of the service (service.yaml), which is essentially the main file...

Reply:
`The service configuration file`

Reply:
Indeed, this matches the naming convention... I missed that one...